design/one-pager-stack-relationship-labels.md

```diff
@@ -0,0 +1,165 @@
+  labels:
+    core.crossplane.io/parent-kind: "stack.stacks.crossplane.io"
+    core.crossplane.io/parent-name: "sample-stack-wordpress"
+    core.crossplane.io/parent-namespace: "app-project1-dev"
+    app.kubernetes.io/managed-by: stack-manager
+...
+```
+
+### Example wordpress `resource.yaml`
+
+```yaml
+apiVersion: wordpress.samples.stacks.crossplane.io/v1alpha1
+kind: WordpressInstance
+metadata:
+  annotations:
+    kubectl.kubernetes.io/last-applied-configuration: |
+      {
+        "apiVersion":"wordpress.samples.stacks.crossplane.io/v1alpha1",
+        "kind":"WordpressInstance",
+        "metadata":{
+          "annotations":{},
+          "name":"my-wordpressinstance",
+          "namespace":"app-project1-dev"
+        }
+      }
+  creationTimestamp: "2019-10-23T23:47:40Z"
+  generation: 1
+  name: my-wordpressinstance
+  namespace: app-project1-dev
+  resourceVersion: "6445"
+  selfLink: /apis/wordpress.samples.stacks.crossplane.io/v1alpha1/namespaces/app-project1-dev/wordpressinstances/my-wordpressinstance
+  uid: f2d13a15-1f9b-40a7-a173-a40abefa61bf
+  labels:
+    core.crossplane.io/parent-kind: "customresourcedefinition.apiextensions.k8s.io"
+    core.crossplane.io/parent-name: "sample-stack-wordpress"
+    core.crossplane.io/parent-namespace: "app-project1-dev"
+    app.kubernetes.io/managed-by: stack-manager
+...
+```
+
+### Example wordpress `kubernetesapplication.yaml`
+
+```yaml
+apiVersion: workload.crossplane.io/v1alpha1
+kind: KubernetesApplication
+metadata:
+  name: wordpress-app-wordpress
+  labels:
+    stack: sample-stack-wordpress
+    core.crossplane.io/parent-kind: "wordpressinstances.wordpress.samples.stacks.crossplane.io"
+    core.crossplane.io/parent-name: "my-wordpressinstance"
+    core.crossplane.io/parent-namespace: "app-project1-dev"
+    app.kubernetes.io/managed-by: stack-manager
+...
+```
+
+### Example wordpress `kubernetescluster.yaml`
+
+```yaml
+apiVersion: compute.crossplane.io/v1alpha1
+kind: KubernetesCluster
+metadata:
+  name: wordpress-cluster-wordpress
+  labels:
+    stack: sample-stack-wordpress
+    core.crossplane.io/parent-kind: "wordpressinstances.wordpress.samples.stacks.crossplane.io"
+    core.crossplane.io/parent-name: "my-wordpressinstance"
+    core.crossplane.io/parent-namespace: "app-project1-dev"
+    app.kubernetes.io/managed-by: stack-manager
+...
+```
+
+### Example wordpress `mysqlinstance.yaml`
+
+```yaml
+apiVersion: database.crossplane.io/v1alpha1
+kind: MySQLInstance
+metadata:
+  name: wordpress-mysql-wordpress
+  labels:
+    stack: sample-stack-wordpress
+    core.crossplane.io/parent-kind: "wordpressinstances.wordpress.samples.stacks.crossplane.io"
+    core.crossplane.io/parent-name: "my-wordpressinstance"
+    core.crossplane.io/parent-namespace: "app-project1-dev"
+    app.kubernetes.io/managed-by: stack-manager
+...
+```
+
+As per the above file example snippets, as long as the labels are defined on children CRD's,
```

Comment:
I would probably use `objects` instead of `CRDs` here, since we're talking about finding any object type, not just CRDs. CRD typically refers to the definition itself, so in this context, that objects like the one which came from `crd.yaml` in this document.